ballista/client/src/context.rs

```diff
@@ -462,12 +462,15 @@ impl BallistaContext {
 }
 
 #[cfg(test)]
-mod tests {
-    #[cfg(feature = "standalone")]
+#[cfg(feature = "standalone")]
+mod standalone_tests {
```

Comment:
Drive-by cleanup to declare that the whole module requires the `standalone` feature rather than declare this on each import and each test.